pkgs/by-name/qu/quasar/package.nix

```
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  alsa-lib,
+  xorg,
+  libGL,
+  freetype,
+  ninja,
+  python3,
+  pkg-config,
+  jack1,
+}:
+
+stdenv.mkDerivation {
+  pname = "quasar";
+  version = "1.0.6";
+
+  src = fetchFromGitHub {
+    owner = "DarkRTA";
+    repo = "quasar";
+    rev = "11e24bb569ef914e1a9641323a2b9edd5b0b3e95";
+    hash = "sha256-nNGaNFUIZp+rP6K6fbg+HnWpev5M44K47MGapTAyvPE=";
+    fetchSubmodules = true;
+  };
+
+  configurePhase = ''
+    for f in lv2 vst2 vst3 standalone; do
+      python build-scripts/configure.py $f
+      mv build.ninja $f.ninja
+    done
+  '';
+
+  buildPhase = ''
+    for f in lv2 vst2 vst3 standalone; do
+      ninja -f $f.ninja
+    done
+  '';
```

Phases must remain adjustable
```suggestion
  configurePhase = ''
    runHook preConfigure
    for f in lv2 vst2 vst3 standalone; do
      python build-scripts/configure.py $f
      mv build.ninja $f.ninja
    done
    runHook postConfigure
  '';

  buildPhase = ''
    runHook preBuild
    for f in lv2 vst2 vst3 standalone; do
      ninja -f $f.ninja
    done
    runHook postBuild
  '';
```